ironfish/src/rpc/clients/client.ts

```diff
@@ -365,6 +367,15 @@ export abstract class RpcClient {
       )
     },
 
```

Comment:
Not in love with this "follow" naming pattern, but I see it predates this PR, so I am fine leaving or changing it.

Reply:
I used `follow` here to distinguish from the streaming `getTransactions` endpoint that's already here for the mempool.

I'm open to renaming!

Reply:
I like `getTransactonsStream`, we use that naming in `/chain`